analysis_runner/cli.py

```diff
@@ -63,21 +67,36 @@ def main(
     _repository = repository
     _commit_ref = commit
     _script = list(script)
+    _cwd = None
+
     if ' ' in _script[0]:
         _script = _script[0].split() + _script[1:]
 
     # false-y value catches empty list / tuple as well
     if not _script:
         _script = ['main.py']
 
+    if _script[0]:
+        if os.path.exists(_script[0]):
+            _perform_shebang_check(_script[0])
+        elif not which(_script[0]):
+            # the first el of _script is not executable
+            # (at least on this computer)
+            if not confirm_choice(
+                f"The first element of the script '{_script[0]}' was not executable \n"
```

Comment:
When would this occur except for the file not being found? If it's only the latter, maybe the error message could be simplified a bit.

Reply:
This handles the case like `analysis-runner pythn3 <your-script>`, where the first element (a) is not a file; (b) is not an executable. Maybe you misspelled the executor, or some other reason, it's just there to confirm. Maybe it makes sense to add an option like (--skip-checks) to skip file-existence or executor checks like this - but can evaluate that later.

Reply:
No you don't, and that's why it confirms it :) And maybe why it might make sense to add a `--skip-checks` flag.